### PR TITLE
Blog post about whats new in 4.1 release. WIP

### PR DESCRIPTION
**Pull request by davsclaus**
Feedback and updates welcome

### CODE REVIEW

File: content/blog/2023/10/camel41-whatsnew/index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+preview: Details of what we have done in the Camel 4.1 release.
+---
+
+Apache Camel 4.1 (non LTS) has just been [released](/blog/2023/10/RELEASE-4.1.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+## DSL
+
+The XML and YAML DSL now has better support for defining <bean> which can be configured with properties, and
```
Comment (gzurowski) [inline content/blog/2023/10/camel41-whatsnew/index.md:15]:
```suggestion
The XML and YAML DSL now has better support for defining `<bean>` which can be configured with properties, and
```

File: content/blog/2023/10/camel41-whatsnew/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+The `rest` component can now use a custom `HeaderFilterStrategy`.
+
+The HTTP server based components will now mute exceptions by default (avoid sending stactraces in HTTP 500 errors back to clients).
+Set `muteException=false` to have old behaviour.
+
+## New Components
+
+- camel-aws2-redshift-data - Perform operations on AWS Redshift using Redshift Data API.
+- camel-aws2-timestream - A Camel Amazon Web Services TimeStream component
+- camel-thymeleaf - Transform messages using a Thymeleaf template.
```
Comment (gzurowski) [inline content/blog/2023/10/camel41-whatsnew/index.md:85]:
```suggestion
- `camel-aws2-redshift-data` - Perform operations on AWS Redshift using Redshift Data API.
- `camel-aws2-timestream` - A Camel Amazon Web Services TimeStream component
- `camel-thymeleaf` - Transform messages using a Thymeleaf template.
```

File: content/blog/2023/10/camel41-whatsnew/index.md (hunk 1 above)
Comment (davsclaus) [reply content/blog/2023/10/camel41-whatsnew/index.md:15]: This causes build failure when using bean in quotes

Comment (gzurowski) [inline content/blog/2023/10/camel41-whatsnew/index.md:15]:
```suggestion
The XML and YAML DSL now have better support for defining <bean> which can be configured with properties, and
```

File: content/blog/2023/10/camel41-whatsnew/index.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+## DSL
+
+The XML and YAML DSL now has better support for defining <bean> which can be configured with properties, and
+references to other beans. We have added support for easier configuration to specify constructor arguments,
+factory beans and methods, and inlined script or java code that gives full power for creating the bean.
+
+We have also enhanced `camel-core` to make it able to dump route DSLs with <bean> included. This makes it possible
+for Camel being able to migrate your route sources between XML and YAML DSLs (see more in next section).
+
+The XML DSL `camel-xml-io` now have line-precise parsing error that shows you exactly where the problem is.
```
Comment (gzurowski) [inline content/blog/2023/10/camel41-whatsnew/index.md:22]:
```suggestion
The XML DSL `camel-xml-io` now has line-precise parsing error that shows you exactly where the problem is.
```

File: content/blog/2023/10/camel41-whatsnew/index.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+## Camel Kafka
+
+Added type converter for `byte[]` to `String` to make it easy to get Kafka header content as text value
+(data is stored as byte array by Kafka).
+
+## Camel OpenTelemtry
+
+Added `OpenTelematryTracingStrategy` that allows fine-grained tracing of every step a message is routed by Camel.
+
+Decorators now uses [OpenTelemtry Sematic Naming Style](https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/)
```
Comment (gzurowski) [inline content/blog/2023/10/camel41-whatsnew/index.md:55]:
```suggestion
Decorators now uses [OpenTelemetry Semantic Naming Style](https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/)
```

File: content/blog/2023/10/camel41-whatsnew/index.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+## Spring Boot
+
+Upgraded to latest 3.1.4 release.
+
+## Miscellaneous
+
+The `camel-report:route-coverage` Maven plugin can now generate reports in HTML format.
+
+The `file` component now have options to accept hidden files and directories.
```
Comment (gzurowski) [inline content/blog/2023/10/camel41-whatsnew/index.md:66]:
```suggestion
The `file` component now has options to accept hidden files and directories.
```

**Review by gzurowski — APPROVED** [review]
LGTM 👍

**Comment by orpiske** [comment]
Hey Claus, this release also brings the new type converter. We can something like: "this release brings the new type converter, that works around the JDK issue 8180450 and can bring improved performance for many scenarios (notably, the content-based router and filter, should have better throughput with this release)."

Obs: i am on the road, so can't elaborate more.

**Pull request merged**